extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -292,4 +286,19 @@ public function onPageHeaderDropdownActions( array &$actions ) {
 
 		return true;
 	}
+
+	/**
+	 * @param \RequestContext $context
+	 * @param $types
+	 * @return bool
+	 */
+	private function shouldHideEditorForInfoboxBuilder( \RequestContext $context, $types ) {
+		global $wgEnablePortableInfoboxBuilderExt;
+
+		return $wgEnablePortableInfoboxBuilderExt
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1067)